### ISSUE TITLE
Received unknown parameter: transfer_data[destination]

### ISSUE THREAD

**Issue by davidkhayutin** [stripe/stripe-node#590]
I'm hoping someone can help me.  I am using stripe in Canada. I am doing a stripe charge without doing a capture. Afterward, when I process the charge I add in the transfer_data[destination] while doing the charge.capture as per the docs. In the docs, it says that in this transfer_data object you can pass in a destination attribute. Yet every time I try I am met with the error

Error: Received unknown parameter: transfer_data[destination]

Do I need to add something to my initial charge object? Not sure why this isn't working! 

I should note the destination are going to stripe connect accounts from my platform

**Comment by remi-stripe** [comment]
@davidkhayutin Today, it is not possible to set `transfer_data[destination]` when capturing a Charge. This is not a supported parameter as you can see here: https://stripe.com/docs/api/charges/capture#capture_charge-transfer_data

**Comment by davidkhayutin** [comment]
is it possible to transfer a charge that has not been captured, like a pre-authorization where capture is set to false

**Comment by remi-stripe** [comment]
@davidkhayutin It is possible using the flow I mentioned above. You would capture the Charge first _and then_ transfer the funds to the connected account.

I'd recommend reaching out to our support team if you have questions about this flow as they would be better equipped to help clarify. Github issues are limited to bugs in the library itself.

You can contact them here: https://support.stripe.com/contact

**Comment by mifi** [comment]
FWIW I got this error when trying to
```js
paymentIntents.update(id, { transfer_data: { destination: 'x' } })
```
because I reused the parameters from the `paymentIntents.create` method when updating the PI (when customer changes their mind and chooses a different product). However it turns out that only the `paymentIntents.create` method supports `destination`. So if we want to change the destination for a payment intent we have to create a new payment intent instead.
